src/js/pages/Escrow/components/CardEscrowSeller.jsx

```diff
@@ -80,8 +68,8 @@ class PreFund extends Component {
       return <Loading page={true}/>; // Wait for trade to be populated
     }
 
-    const enoughBalance = toBN(trade.token.balance ? toTokenDecimals(trade.token.balance, trade.token.decimals) : 0).gte(toBN(trade.tradeAmount)) &&
-                          toBN(toTokenDecimals(tokens.SNT.balance, 18)).gte(toBN(fee));
+    const enoughBalance = toBN(trade.token.balance ? toTokenDecimals(trade.token.balance || 0, trade.token.decimals) : 0).gte(toBN(trade.tradeAmount)) &&
```

Comment:
is it possible for the balance to return undefined or null?

Reply:
Yeah it happened to me sometimes and it would cause a crash. So that makes sure it doesn't crash while the balance is loading.